### DISCUSSION TITLE
分享一下 P.A.R.A. 的最佳实践

### DISCUSSION THREAD

**Discussion by Quorafind** [obsidianzh/forum#39] [实用技巧/最佳实践]
近来，在工作中发现需要经常建立单日或者单周的项目文件或领域文件，而之前又接触了一些关于 P.A.R.A. 的资料，发现它很适合用在工作中，但是如果将 MOC 模式切换到 P.A.R.A 显然会有比较长时间的阵痛，所以有没有 P.A.R.A. 的最佳实践来减少阵痛的时间，请分享一下。

**Comment by dearxs** [comment]
什么是 MOC

**Reply by makerjackie** [reply]
map of content 也就是 **笔记地图**，简单来说，就是建立一个 `home主页` 然后主页里放一堆相关的链接，充当 笔记导航 / 目录的功能。

**Reply by dearxs** [reply]
了解了，谢谢。（就是目录的意思吧。）

**Reply by makerjackie** [reply]
对，差不多，自定义目录，还可以在目录里加上标签 搜索关键词的索引
